recipes/R/Retail - GradientBoosting/README.md

```diff
@@ -16,6 +34,9 @@ sh ./build.sh
 Please note the `login.sh` script should only need to be run once.
 
 
+### Video for Training, Scoring and Saving data
+[![Watch the video](../../docs/images/HomePage.png)](https://youtu.be/rur0jkqhvno)
```

Comment:
Is it possible to have collateral assets like this owned by an Adobe-owned entity?  (e.g. Adobe's Platform Youtube account if it exists)

Reply:
Sure, please let me know which account to use.